### DISCUSSION TITLE
Can Oh My Posh work with Cmd w/o Powershell?

### DISCUSSION THREAD

**Discussion by cardin** [JanDeDobbeleer/oh-my-posh#3497] [Troubleshoot] [answered]
My computer can't use Powershell (it's been disabled at the Administrator level via Group Policy).

Is it still possible for me to use Oh My Posh with cmd and clink?

The reason I ask is, the download step itself seems to require Powershell?

**Comment by cardin** [comment, marked as answer]
Nevermind, I tried to install Oh My Posh via Windows Store, got the portable version of Clink, and added a .lua script as described in the Oh My Posh docs.

**It works - no Powershell needed.**

**Reply by 0PandaDEV** [reply]
Where is the clink script folder located and what line did you use because I can't get it to work.

**Reply by JanDeDobbeleer** [reply]
@0PandaDEV you can find that information in the [docs](https://ohmyposh.dev/docs/installation/prompt).

**Reply by 0PandaDEV** [reply]
No i cant find any information online on where the script directory is.

**Reply by JanDeDobbeleer** [reply]
<img width="968" alt="image" src="https://github.com/JanDeDobbeleer/oh-my-posh/assets/2492783/e1676c49-063d-45ee-93a5-184722cd7cdc">

**Reply by 0PandaDEV** [reply]
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/70103896/e31753bf-8370-4f02-ae44-1101f13afdd8)

**Reply by JanDeDobbeleer** [reply]
@0PandaDEV wrong clink, you have installed the ancient, non-supported version. [This](https://chrisant996.github.io/clink/) is what you need, that link is also in the docs.

**Reply by 0PandaDEV** [reply]
oh thanks
